exercises/binary-search-tree/binary_search_tree.py

```diff
@@ -0,0 +1,14 @@
+class TreeNode(object):
+    def __init__(self, value):
+        self.value = value
+
+
+class BinarySearchTree(object):
+    def __init__(self):
+        pass
+
+    def add(self, value):
+        pass
+
+    def search(self, search_number):
```

Comment:
I think this should use `value` rather than `search_number` as an argument, since we put values in and then search for them, and `search_` is probably redundant.

Reply:
Done!

Reply:
@rodrigocam, it looks like you've updated this in `example.py` but not here.

Reply:
Ok, done!